source/Makefile

```diff
@@ -764,10 +770,11 @@ endef # multi_lang_rule
 $(foreach group_code,$(LANGUAGE_GROUPS),$(eval $(call multi_lang_rule,$(group_code),$(LANGUAGE_GROUP_$(group_code)_NAME),$(LANGUAGE_GROUP_$(group_code)_LANGS))))
 
 
-clean :
+clean:
 	rm -Rf Core/Gen
 	rm -Rf $(OUTPUT_DIR_BASE)
-	rm -Rf $(HEXFILE_DIR)/*
+	rm -Rf $(HEXFILE_DIR)

```

Comment:
Please dont rm the Hexfile folder if we can, it makes it far more annoying as it means file mangers will kick you out (as the folder is gone) or a terminal in the folder will have a broken path.

Reply:
Oh, wait a minute... I sincerely sorry to bother you, just expressing some suggestion before updating PR - how about adding another one target, something like this right next to `clean` target:
```
mrproper / clean-all: clean
    rmdir $(HEXFILE_DIR)
```
Just curious what do you think about such approach? Thanks.

Reply:
`clean-all` would be totally fine by me :) 😁 🤗 


I totally get you on having a full cleanup; I probably should throw a comment on it as to why that folder gets left hanging around. It was deliberate, but probably not obvious until one does a few device bring ups 🤣